### PR TITLE
registryworkshop: Add version 5.0.1

### PR DESCRIPTION
**Pull request by wenmin92**
[Registry Workshop](http://www.torchsoft.com/en/rw_information.html) is an advanced registry editor. It is a perfect replacement for RegEdit and RegEdt32 which shipped with Windows.

### CODE REVIEW

File: bucket/registryworkshop.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "shortcuts": [
+                [
+                    "RegWorkshopX64.exe",
+                    "Registry Workshop"
+                ]
+            ]
+        },
+        "32bit": {
+            "bin": [
```
Comment (Ash258) [inline bucket/registryworkshop.json]:
```suggestion
            "bin": "RegWorkshop.exe",
```

File: bucket/registryworkshop.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                [
+                    "RegWorkshop.exe",
+                    "Registry Workshop"
+                ]
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\RegWorkshop.ini\")) { New-Item \"$dir\\RegWorkshop.ini\" -ItemType File > $Null}",
+        "if (!(Test-Path \"$persist_dir\\favorites.dat\")) { New-Item \"$dir\\favorites.dat\" -ItemType File > $Null}"
```
Comment (Ash258) [inline bucket/registryworkshop.json]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\favorites.dat\")) { New-Item \"$dir\\favorites.dat\" -ItemType File | Out-Null}"
```

File: bucket/registryworkshop.json [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            "shortcuts": [
+                [
+                    "RegWorkshop.exe",
+                    "Registry Workshop"
+                ]
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\RegWorkshop.ini\")) { New-Item \"$dir\\RegWorkshop.ini\" -ItemType File > $Null}",
```
Comment (Ash258) [inline bucket/registryworkshop.json]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\RegWorkshop.ini\")) { New-Item \"$dir\\RegWorkshop.ini\" -ItemType File | Out-Null }",
```

File: bucket/registryworkshop.json (hunk 2 above)
Comment (wenmin92) [reply bucket/registryworkshop.json]: [null - What's the better (cleaner) way to ignore output in PowerShell? - Stack Overflow](https://stackoverflow.com/questions/5260125/whats-the-better-cleaner-way-to-ignore-output-in-powershell)

Comment (wenmin92) [reply bucket/registryworkshop.json]: Why is this bad?

**Comment by Ash258** [comment]
I do not think this should be in official bucket

http://www.torchsoft.com/en/download.html

> Registry Workshop (Free **30-Day Fully Functional Trial**)

File: bucket/registryworkshop.json (hunk 2 above)
Comment (Ash258) [reply bucket/registryworkshop.json]: No reason to use bash like style of redirection output. Powershell is using pipes, pipes are used in all code base, 

Just nonsense to save few milliseconds.

Comment (wenmin92) [reply bucket/registryworkshop.json]: OK, accept.

**Comment by wenmin92** [comment]
I think scoop can manage a wide variety of applications, including paid apps.

**Comment by wenmin92** [comment]
[App Manifests · lukesampson/scoop Wiki](https://github.com/lukesampson/scoop/wiki/App-Manifests)
> For other licenses, use the URL of the license, if available. Otherwise, use “Freeware”, “Proprietary”, “Public Domain”, “Shareware”, or “Unknown”, as appropriate.

**Comment by wenmin92** [comment]
Well, you may right, there should be a special bucket for these paid apps.

**Pull request closed without merging**